<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ePLIbidem</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../app.ico</normaloff>../app.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitterVertical">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QFrame" name="FrameLeft">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="NoteBrowser">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <column>
          <property name="text">
           <string>Nota</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Texto</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="FrameRight">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitterHorizontal">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>3</number>
         </property>
         <widget class="QFrame" name="IbidFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="NoteLayout">
             <item>
              <widget class="QLabel" name="NoteLabel">
               <property name="text">
                <string>Nota:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="NoteEntry">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="NoteIdLabel">
               <property name="text">
                <string>Id:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="NoteIdEntry">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="NoteToIbidButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Pasar nota a ibíd.</string>
               </property>
               <property name="text">
                <string>A ibid</string>
               </property>
               <property name="icon">
                <iconset theme="format-indent-more">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="NoteSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="FontIncreaseSizeButton">
               <property name="toolTip">
                <string>Aumenta el tamaño del texto.</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="icon">
                <iconset theme="format-font-size-more">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="shortcut">
                <string>Ctrl++</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="FontDecreaseSizeButton">
               <property name="toolTip">
                <string>Disminuye el tamaño del texto.</string>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="icon">
                <iconset theme="format-font-size-less">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="shortcut">
                <string>Ctrl+-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="TagButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Aplica/Oculta estilos HTML.
(No se puede editar la nota en este modo)</string>
               </property>
               <property name="text">
                <string>Tag HTML</string>
               </property>
               <property name="icon">
                <iconset theme="format-text-code">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="NoteCurrent">
               <property name="text">
                <string>1 de 1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="NotePrevButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Retrocede a la nota anterior.</string>
               </property>
               <property name="text">
                <string>Anterior</string>
               </property>
               <property name="icon">
                <iconset theme="go-previous">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="NoteNextButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Avanza a la siguiente nota.</string>
               </property>
               <property name="text">
                <string>Siguiente</string>
               </property>
               <property name="icon">
                <iconset theme="go-next">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="NoteText">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Texto de la nota base.</string>
             </property>
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="NoteHrefLayout">
             <item>
              <widget class="QLabel" name="NoteHrefLabel">
               <property name="text">
                <string>href:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="NoteHrefEntry">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="NoteFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="IbidControlsLayout">
             <item>
              <widget class="QLabel" name="IbidLabel">
               <property name="text">
                <string>Ibíd:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="IbidEntry">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="IbidIdLabel">
               <property name="text">
                <string>Id:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="IbidIdEntry">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="IbidToNoteButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Pasar ibíd. a nota.</string>
               </property>
               <property name="text">
                <string>A  nota</string>
               </property>
               <property name="icon">
                <iconset theme="format-indent-less">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="IbidUndoButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Restaura la nota al texto original.</string>
               </property>
               <property name="text">
                <string>Deshacer</string>
               </property>
               <property name="icon">
                <iconset theme="edit-undo">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="shortcut">
                <string>Ctrl+U</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="IbidSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="ShowOriginalIbid">
               <property name="toolTip">
                <string>Muestra/Oculta el ibid original.</string>
               </property>
               <property name="text">
                <string>Ver original</string>
               </property>
               <property name="icon">
                <iconset theme="view-visible">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="IbidCurrent">
               <property name="text">
                <string>1 de 1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="IbidPrevButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Retrocede a la nota ibíd. anterior.</string>
               </property>
               <property name="text">
                <string>Anterior</string>
               </property>
               <property name="icon">
                <iconset theme="go-previous">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="shortcut">
                <string>Ctrl+PgUp</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="IbidNextButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Avanza a la siguiente nota ibíd.</string>
               </property>
               <property name="text">
                <string>Siguiente</string>
               </property>
               <property name="icon">
                <iconset theme="go-next">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="shortcut">
                <string>Ctrl+PgDown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="IbidOriginalText">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Texto de la nota ibíd. original.</string>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="IbidText">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Texto de la nota ibíd.</string>
             </property>
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="IbidHrefLayout">
             <item>
              <widget class="QLabel" name="IbidHrefLabel">
               <property name="text">
                <string>href:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="IbidHrefEntry">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="IbidModifyLayout">
             <item>
              <widget class="QToolButton" name="ConfigButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Configuración del procesado.</string>
               </property>
               <property name="text">
                <string>Conf.</string>
               </property>
               <property name="icon">
                <iconset theme="application-menu">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="IbidProcessButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Aplica la RegEx y las opciones seleccionadas a la nota.
Los cambios serán descartados si no se pulsa el botón Reemplazar.</string>
               </property>
               <property name="text">
                <string>Procesar Ibíd</string>
               </property>
               <property name="shortcut">
                <string>Ctrl+Backspace</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="IbidProcessAllButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Procesa todas las notas y las reemplaza.</string>
               </property>
               <property name="text">
                <string>Procesar y reemplazar todo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="IbidReplaceButton">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>Reemplaza el texto de la nota.
Si no se reemplaza los cambios en la nota se descartarán.</string>
               </property>
               <property name="text">
                <string>Reemplazar</string>
               </property>
               <property name="shortcut">
                <string>Ctrl+Return</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="BotomLayout">
     <item>
      <widget class="QLabel" name="Messenger">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Ibid que no han sido modificados</string>
       </property>
       <property name="text">
        <string>Ibíd sin modificar: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="messageSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AcceptButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Confirma los cambios y escribe el archivo.</string>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Cancela todos los cambios y no modifica el archivo.</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
